packages/viewtoggle/package.json

```diff
@@ -9,21 +9,22 @@
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "scripts": {
-    "build": "run-s build:js build:cjs build:css",
-    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
+    "build": "run-s build:svg build:cjs build:esm build:css copy:svg",
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
-    "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
-    "build:watch": "npm run build:js -- --watch",
+    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
+    "build:svg": "svgr --typescript --filename-case kebab --ext \"dist.tsx\" -d src/react/illustrations src/svg",
```

Comment:
Looks like a copy-pasta mistakenelli to me. This package doesn't have svg.

Reply:
thanks. yes i copied from emptystate 😬